packages/pds/src/sequencer/events.ts

```diff
@@ -0,0 +1,144 @@
+import Database from '../db'
+import { z } from 'zod'
+import { cborEncode, schema } from '@atproto/common'
+import {
+  BlockMap,
+  blocksToCar,
+  CidSet,
+  CommitData,
+  WriteOpAction,
+} from '@atproto/repo'
+import { PreparedWrite } from '../repo'
+import { CID } from 'multiformats/cid'
+
+export const sequenceCommit = async (
+  dbTxn: Database,
+  did: string,
+  commitData: CommitData,
+  writes: PreparedWrite[],
+) => {
+  let tooBig: boolean
+  const ops: CommitEvtOp[] = []
+  const blobs = new CidSet()
+  let carSlice: Uint8Array
+
+  // max 200 ops or 1MB of data
+  if (writes.length > 200 || commitData.blocks.byteSize > 1024000) {
```

Comment:
Such a goofy nit, sorry! 🙈 But I wonder if we should either go for 1000^2 or 1024^2 for the max byte size, rather than 1024*1000.  Realize you may have had a good reason for that I'm overlooking though!